db/migrate/20201023124124_add_person_duplicates.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2020, Jungwacht Blauring Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class AddPersonDuplicates < ActiveRecord::Migration[6.0]
+  def change
+    create_table(:person_duplicates) do |t|
+      t.integer :person_1_id, null: false
+      t.integer :person_2_id, null: false
+      t.boolean :acknowledged, null: false, default: false
+
+      t.timestamps
+    end
+
+    add_foreign_key(:person_duplicates, :people, column: :person_1_id)
```

Comment:
is this required when you use t.references?
never used add_foreign_key before ...